android/src/main/java/com/rudderstack/sdk/flutter/RudderSdkFlutterPlugin.java

```diff
@@ -52,15 +60,17 @@ public void onAttachedToEngine(
                 );
         channel.setMethodCallHandler(this);
         context = flutterPluginBinding.getApplicationContext();
+        ActivityLifeCycleHandler.registerActivityLifeCycleCallBacks(context);
```

Comment:
Is it okay to track application open and backgrounded even if trackLifeCycleEvents is false?

Reply:
Actually we are checking if `trackLifeCycleEvents` is false or not in the Activity Life Cycle callbacks